ocs_ci/ocs/ocp.py

```diff
@@ -788,6 +788,48 @@ def is_exist(self, resource_name="", selector=None):
             )
             return False
 
+    def get_logs(
+            self,
+            name,
+            container_name=None,
+            all_containers=False,
+            secrets=None,
+            timeout=None,
+            ignore_error=False,
+    ):
+        """
+        Execute ``oc logs`` command to fetch logs for a given k8s resource.
+
+        Since the log is stored as a string in memory, this will be
+        problematic when the log is large.
+
+        Args:
+            name (str): name of the resource to fetch logs from
+            container_name (str): name of the container (optional)
+            all_containers (bool): fetch logs from all containers of the
+                resource
+            secrets (list): A list of secrets to be masked with asterisks
+            timeout (int): timeout for the oc_cmd
+            ignore_error (bool): True if ignore non zero return code and do not
+                raise the exception.
+
+        Returns:
+            str: container logs
+        """
+        log.info("fetching logs from %s/%s", self.kind, name)
```

Comment:
same for using fstring

Reply:
[Our code guidelines](https://ocs-ci.readthedocs.io/en/latest/docs/coding_guidelines.html) suggests to use `%` based logger formatting.

Reply:
ack